Templates/CRMs/HubSpot/HubSpot to JSON - Companies/long-description.md

```diff
@@ -0,0 +1,21 @@
+This template retrieves company data from HubSpot, and downloads it to a local fileshare as a single JSON file.
+
+![Template](assets/HubSpot_to_JSON_-_Companies.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- Using the HubSpot API requires an API key.
```

Comment:
The "Using the HubSpot API requires an API key" is not a prerequisite per se, rather a statement. Something like "HubSpot API key is accessible" is more like a prerequisite in this context.